pkg/config/config.go

```diff
@@ -37,6 +37,8 @@ type ContainerdConfig struct {
 	DefaultRuntime Runtime `toml:"default_runtime" json:"defaultRuntime"`
 	// UntrustedWorkloadRuntime is a runtime to run untrusted workloads on it.
 	UntrustedWorkloadRuntime Runtime `toml:"untrusted_workload_runtime" json:"untrustedWorkloadRuntime"`
+	// NoPivot is runC option for running container in RamDisk
```

Comment:
// NoPivot disables pivot-root (linux only), required when running a container in a RamDisk with runc

Reply:
good, done